<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowkind">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="party_gui_insert_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Party</string>
   </property>
  </widget>
  <widget class="QPushButton" name="party_update_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Weight:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="party_delete_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>delete Party</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="party_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>511</width>
     <height>621</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>5</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit App</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="party_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="ideas_in_table">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>381</width>
     <height>511</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>5</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="ideas_out_table">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>130</y>
     <width>261</width>
     <height>621</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>5</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Partys</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ideas</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="idea_update_button">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>70</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Idea</string>
   </property>
  </widget>
  <widget class="QPushButton" name="idea_delete_button">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>100</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>delete Idea</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Modification PID to:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="idea_insert_button">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Idea</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idea_name">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="party_idea_set_button">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>100</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>add idea to party</string>
   </property>
  </widget>
  <widget class="QPushButton" name="party_idea_del_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Abolish Idea from party</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="ideas_stan_table">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>650</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>5</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="party_weight">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
